xarray/core/dataarray.py

```diff
@@ -256,23 +258,6 @@ def _replace_maybe_drop_dims(self, variable, name=__default):
                                  if set(v.dims) <= allowed_dims)
         return self._replace(variable, coords, name)
 
-    def _replace_indexes(self, indexes):
```

Comment:
Thanks.
Yes, in `DataArray.__getitem__` and also in `DataAarray.isel`, `_to_temp_dataset` and `_from_temp_dataset` are now being used.